viewer/README.md

```diff
@@ -1,2 +1,7 @@
 # XComponent viewer
+
 Web viewer used in xcomponent visual studio code extension
```

Comment:
I replaced by a simple comment.
The readme file of the viewer is not really the main purpose of this pull request (compared to the other readme)